apps/openmw/mwclass/npc.cpp

```diff
@@ -417,6 +419,7 @@ namespace MWClass
     void Npc::insertObjectRendering (const MWWorld::Ptr& ptr, const std::string& model, MWRender::RenderingInterface& renderingInterface) const
     {
         renderingInterface.getObjects().insertNPC(ptr);
+        ptr.getRefData().getBaseNode()->setNodeMask(MWRender::Mask_Actor);
```

Comment:
I assume setNodeMask calls were moved from insertNPC and insertCreature to class classes for consistency with other objects, but is it really necessary? Shouldn't it be fine to assume any inserted creature and actor must have Mask_Actor node mask?